com.kassindornelles.lolforlinuxinstaller.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: com.kassindornelles.lolforlinuxinstaller
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-22.08
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+finish-args:
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+command: com.kassindornelles.lolforlinuxinstaller
+modules:
+    - name: PyQtApp
+    - python3-modules.json
+    - name: lol-for-linux
+      buildsystem: simple
+      cleanup:
+        - '.git'
+        - '.github'
+        - '.gitignore'
+        - 'license.md'
+        - './src'
+        - 'wine_build.json'
+        - 'README.md'
```

Comment:
I'm pretty sure this does't do what you think it does.

this is not `make clean`

Reply:
really?, it didn't said anything about not working with simple-build in the docs, but i changed to cleanup-commmands

Reply:
`cleanup` is to remove installed artifact (like development stuff) and not about `make clean`. The build directory is wipe in every case.

so no.